dotnet/src/SemanticKernel/KernelExtensions/InlineFunctionsDefinitionExtension.cs

```diff
@@ -42,7 +44,8 @@ public static class InlineFunctionsDefinitionExtension
         double topP = 0,
         double presencePenalty = 0,
         double frequencyPenalty = 0,
-        IEnumerable<string>? stopSequences = null)
+        IEnumerable<string>? stopSequences = null,
+        int httpTimeoutInSeconds = OpenAIClientAbstract.DefaultHttpTimeoutInSeconds)
```

Comment:
this is hardcoding the kernel to OpenAI settings, moving in the opposite direction of our design. The kernel needs to be model agnostic, in fact some models might not even require a HTTP request if one is using local inference. For now, let's leave this out.